Docker-Swarm-deployment/single-node/immudb/script.py

```diff
@@ -0,0 +1,46 @@
+import immudb
+from immudb import ImmudbClient
+
+f = open("../run/secrets/adminpassword","r")
+ADMIN_PASSWORD = f.read()
+
+f = open("../run/secrets/authpassword","r")
+IUDX_AUTH_PASSWORD = f.read()
+
+f = open("../run/secrets/catpassword","r")
+IUDX_CAT_PASSWORD = f.read()
+
+f = open("../run/secrets/defaultpassword","r")
+IUDX_DEFAULT_PASSWORD = f.read()
+
+f = open("../run/secrets/rspassword","r")
+IUDX_RS_PASSWORD = f.read()
+
+f = open("../run/secrets/testpassword","r")
+IUDX_TEST_PASSWORD = f.read()
+
+client = ImmudbClient("immudb:3322")
+client.login("immudb","immudb")
+
+client.changePassword("immudb",ADMIN_PASSWORD,"immudb")
+client.login("immudb",ADMIN_PASSWORD)
+
+client.databaseCreate("iudxauth")
+client.databaseCreate("iudxcat")
+client.databaseCreate("iudxrsorg")
```

Comment:
iudxrsorg database, test user, iudx_default user are not needed. 
iudx_rs user has RW permission on iudxrs database. @namanmanish